apps/etterlatte-behandling/src/main/kotlin/sak/SakTilgangDao.kt

```diff
@@ -5,10 +5,13 @@ import no.nav.etterlatte.libs.database.singleOrNull
 import no.nav.etterlatte.libs.database.toList
 import javax.sql.DataSource
 
+/*
+    Hvis denne skal gjøre skriveoperasjoner MÅ ingen av metodene her være wrappet i en intransaction, da vil skriveoperasjonen bli overskridet av transaksjonen til intransaction
```

Comment:
Skjønner ikkje heilt korfor det blir sånn. Er det noko vi kan gjera for å sikre oss? Og: vil det ikkje da feile når vi prøver å opne ein ny transaksjon her fordi det fins ein transaction allereie?

Reply:
Omtrent alle andre dao-ar initialiserer vi med `{ databaseContext().activeTx() }`, men akkurat denne får `DataSource` direkte som parameter. Hadde vore fint om vi i denne og `OppgaveMetrikkerDao` gjorde det likt som i alle dei andre.

Reply:
Tingen er at denne brukes både etter fasen "CALL" der vi har lagt på en databasekontekst men også tidligere i hooken som intercepter rett etter autentisering der man vi ikke har lagt på konteksten vår.
Hvis du tenker at man vil sikre oss mot å bruke den feil vil jeg si at den beste måten er å duplisere tjenesten, en som kan brukes inne i en inTransaction og en som kan brukes uten den.
Men alle operasjonene som er inne i denne i dag er gyldige siden ingen av operasjonene er CRUDE og det vil da fungere fint uansett med mindre man legger til en CRUDE operasjon som blir kalt inne i en `inTransaction` blokk.